binding.cc

```diff
@@ -655,7 +655,7 @@ namespace zmq {
       return Nan::ThrowError("Must pass an option");
     if (!info[0]->IsNumber())
       return Nan::ThrowTypeError("Option must be an integer");
-    int64_t option = Nan::To<int64_t>(info[0]).FromJust();
+    int option = Nan::To<int>(info[0]).FromJust();
```

Comment:
Arg, sorry. Now that I've read it *again*, this is only `option`, which should be an int not int64, and not `optval`, the result which has varying type. Everything is fine here, move along.